### DISCUSSION TITLE
"Difficulty Adjust" mod's sliders are smaller

### DISCUSSION THREAD

**Discussion by Gabixel** [ppy/osu#18919] [Q&A] [answered]
Is this intended?

![image](https://user-images.githubusercontent.com/43073074/176163219-579c0969-cd25-4381-a3c7-78611c1bfb0f.png)

**Comment by peppy** [comment]
scroll bar i guess?

**Reply by Gabixel** [reply]
Apparently, this was also happening in the old design where everything was in one container. So I don't think the scrollbar is the problem:
![image](https://user-images.githubusercontent.com/43073074/176170382-6ce15359-0868-4b12-bcab-27f77a0403c4.png)

**Comment by Gabixel** [comment, marked as answer]
A simple inspection with the Draw Visualiser reveals that the sliders containers have padding applied to them
![image](https://user-images.githubusercontent.com/43073074/176180925-10b3e001-9eff-403b-9853-345c0888033f.png)

**Reply by Joehuu** [reply]
Will be fixed in https://github.com/ppy/osu/pull/18922.
